core/imgread/common.cpp

```diff
@@ -131,7 +131,7 @@ bool ConvertSector(u8* in_buff , u8* out_buff , int from , int to,int sector)
 
 Disc* OpenDisc(const wchar* fn)
 {
-	Disc* rv = nullptr;
+	Disc* rv;
```

Comment:
This should be initialized to 0 `Disc* rv = 0;` to avoid changing functionality.

Reply:
Done